ui/components/app/edit-gas-fee-popover/network-statistics/network-statistics.js

```diff
@@ -32,7 +32,8 @@ const NetworkStatistics = () => {
       <Typography
         color={COLORS.TEXT_ALTERNATIVE}
         fontWeight={FONT_WEIGHT.BOLD}
-        margin={[3, 0]}
+        marginTop={3}
+        marginBottom={3}
         variant={TYPOGRAPHY.H8}
       >
         {t('networkStatus')}
```

Comment:
Updating previous array props for different directions to singular direction props now that array props are reserved for responsive props